docs/decisions/ADR-002-모듈러-모놀리식에서-마이크로서비스로-전환.md

```
@@ -99,15 +64,16 @@ nextmall/
 │   └── bff/
 └── services/                  # 배포 단위
     ├── api-gateway/           # WebFlux
-    ├── bff-service/           # MVC
-    ├── user-service/          # MVC    
+    ├── bff-service/           # WebFlux
+    ├── user-service/          # MVC
     └── auth-service/          # MVC
 ```
 
 **핵심 아이디어**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)